Sources/Identity/IdentityManager.swift

```diff
@@ -50,10 +51,14 @@ class IdentityManager: CurrentUserProvider {
             Logger.warn(Strings.identity.logging_in_with_empty_appuserid)
         }
 
-        let appUserID = appUserID?.notEmptyOrWhitespaces
+        let appUserID = if systemInfo.dangerousSettings.uiPreviewMode {
+            Self.uiPreviewModeAppUserID
```

Comment:
Maybe we should add a debug log indicating that we're defaulting to this specific user ID (and ignoring any given user ID) just in case?